web/src/components/common/AppCard.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-card
+    class="mb-5 simple-card"
+    color="#fff"
+    variant="flat"
+    style="border: 1px solid #d7cfbe"
+  >
+    <v-card-title
+      v-if="title"
+      class="d-flex"
+      :class="{ 'mb-n3': subtitle }"
+    >
+      {{ title }}
+      <v-spacer />
+      <slot name="rightTitle"></slot>
```

Comment:
Consider matching the pattern (and casing) that Vuetify uses for naming this slot.
e.g.
title-append or something like that.

Reply:
Also be aware that Vuetify 3 provide a #title and #subtitle slots, see https://vuetifyjs.com/en/api/v-card/#slots